pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -19,10 +19,13 @@ limitations under the License.
 package attachdetach
 
 import (
+	"context"
 	"fmt"
 	"net"
 	"time"
 
+	"github.com/go-logr/logr"
```

Comment:
If you use klog.Logger instead of logr.Logger, then you don't need this additional import. Also applies elsewhere.

Reply:
ok